lib/HttpClient.php

```diff
@@ -15,36 +15,38 @@ public function __construct($baseUrl, $apiKey, $apiVersion)
         $this->apiVersion = $apiVersion;
     }
 
-    public function get($path, $params = array())
+    public function get($path, $params = [])
```

Comment:
These are functionally equivalent, but `[]` seems to be preferred over `array()`.